prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1888,7 +1911,7 @@ void rxr_ep_progress_internal(struct rxr_ep *ep)
 		if (rx_entry->state == RXR_RX_FREE)
 			continue;
 
-		dlist_remove(&rx_entry->queued_entry);
+		dlist_remove(&rx_entry->queued_ctrl_entry);
```

Comment:
Do you need to &= ~RXR_RX_QUEUED_CTRL here?

Reply:
There is queued_ctrl is a state, not a flag. here we change the state to RXR_RX_RECV